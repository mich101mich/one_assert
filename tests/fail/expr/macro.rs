fn main() {
    one_assert::assert!(dbg!(1));
}
