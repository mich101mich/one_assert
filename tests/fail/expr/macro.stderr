error[E0308]: mismatched types
 --> tests/fail/expr/macro.rs:2:25
  |
2 |     one_assert::assert!(dbg!(1));
  |                         ^^^^^^^ expected `bool`, found integer
  |
  = note: this error originates in the macro `dbg` (in Nightly builds, run with -Z macro-backtrace for more info)
